openslides/users/views.py

```diff
@@ -153,10 +153,11 @@ def destroy(self, request, *args, **kwargs):
     @detail_route(methods=['post'])
     def reset_password(self, request, pk=None):
         """
-        View to reset the password (using the default password).
+        View to reset the password using the requested password.
         """
         user = self.get_object()
-        user.set_password(user.default_password)
+        user.set_password(request.data['password'])
```

Comment:
Can we set the new password to the database default password if the client sends no new password?